### PR TITLE
Week5

### PR DESCRIPTION
**Pull request by AbeFernandez**
Pull for the first time week 5 assignment to "main."

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  width: 75vw;
+  justify-content:space-around;
+}
+
+.card-container{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.card-controls{
```
Comment (panzerama) [inline client/src/App.css:13]: Be consistent in your spacing and formatting.

File: client/src/App.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import './App.css';
+import Topbar from './components/Topbar/Topbar';
+import CardNavegation from './components/CardNavegation/CardNavegation';
+import React from 'react';
```
Comment (panzerama) [inline client/src/App.js]: It isn't something I'll grade you down on, but it is worth your while to organize your imports. Start with any outside libraries and then internal references.

File: client/src/components/CardNavegation/CardNavegation.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.card-nav{
+    width: 10vw;
+    height: 200px;
+    background-color: cadetblue;
+    margin-left: 50px;
+    margin-top: 50px;
+}
+.card-nav li{
```
Comment (panzerama) [inline client/src/components/CardNavegation/CardNavegation.css:9]: Needs consistent spacing

**Review by panzerama — CHANGES_REQUESTED** [review]

File: client/src/App.css (hunk 1 above)
Comment (AbeFernandez) [reply client/src/App.css:13]: I fixed spacing and formating

File: client/src/App.js (hunk 2 above)
Comment (AbeFernandez) [reply client/src/App.js]: I fixed the organization of the imports hopes it looks better now.

File: client/src/components/CardNavegation/CardNavegation.css (hunk 3 above)
Comment (AbeFernandez) [reply client/src/components/CardNavegation/CardNavegation.css:9]: I fixed the spacing of this css file. Hopes it looks better now

**Pull request merged**